synapse/tools/changelog.py

```diff
@@ -210,6 +219,8 @@ def makeargparser():
                              help='Version number')
     format_pars.add_argument('-d', '--date', action='store', type=str,
                              help='Date to use with the changelog entry')
+    format_pars.add_argument('-r', '--rm', default=False, action='store_true',
```

Comment:
Probably want to specify the type here.

Reply:
`action='store_true'` stores a `True` so we don't need a type.